packages/headless/src/api/search/search-api-error-response.ts

```diff
@@ -5,6 +5,7 @@ export interface SearchAPIErrorWithStatusCode {
   statusCode: number;
   message: string;
   type: string;
+  ignore?: boolean;
```

Comment:
Unless we treat this object as a parameter/request/option rather than a response, I would prefer `ignored` over `ignore`.